<?xml version="1.0" encoding="UTF-8"?>
<launch>
    <!-- This version uses the openai_ros environments -->
    <rosparam command="load" file="$(find robot_training)/config/robot_qlearn_params.yaml" />
    <!-- Launch the training system -->

    <node pkg="robot_training" name="my_robot_training_env" type="start_qlearning.py" output="screen"/>
</launch>
